modules/openapi/component-protocol/scenarios/home-page-sidebar/components/myApplication/render.go

```diff
@@ -60,6 +60,7 @@ func (this *MyApplication) getAppsNum(orgID string, queryStr string) (int, error
 		PageSize: 1,
 		PageNo:   1,
 		Query:    queryStr,
+		IsSimple: true,
```

Comment:
what is this `IsSimple`?

Reply:
if Issimple is true, it will not query additional app info like workspace, cluster info, git repo and runtime count.

Reply:
where to use? I did not found the usage

Reply:
where to use? I did not found the usage

Reply:
sorry, I found that it's just query params.